tests/build/build_templates.go

```diff
@@ -155,37 +158,25 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				var pipelineRun *v1beta1.PipelineRun
 
 				BeforeAll(func() {
-					// Create an Service account and a token associating it with the service account
-					resultSA := "tekton-results-tests"
-					_, err := kubeadminClient.CommonController.CreateServiceAccount(resultSA, testNamespace, nil)
+					// create the proxyplugin for tekton-results
+					_, err = kubeadminClient.CommonController.CreateProxyPlugin("tekton-results", "toolchain-host-operator", "tekton-results", "tekton-results")
```

Comment:
ok I now fully understand your query in slack earlier about testing both paths vs. testing just the toolchain path

so yeah in infra-deployments, you have the option of bootstrapping with or without the toolchain stuff

given this is the build-service suite, where the build-service is owned by the build team, you need @prietyc123 to solicit the build team, like @Michkov or @mmorhun, and not just the pipeline service folks like myself or @sayan-biswas , to see if they are OK with requiring the toolchain stuff for the build-service e2e's.  It may be that when they run the tests locally, they would prefer to not have to bootstrap the toolchain stuff.

If that is the case, then yeah, we would need this new flavor in a separate ginkgo describe section etc. so they could filter out toolchain dependent tests.

Let's see what they say.

Reply:
> If that is the case, then yeah, we would need this new flavor in a separate ginkgo describe section etc. so they could filter out toolchain dependent tests.

Hmm, make sense 👍 But IIUC it should not affect build team because we are running the result test in different [when spec](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/build/build_templates.go#L153-L226) @Michkov @mmorhun @psturc @rhopp correct me if I am wrong. [Here](https://issues.redhat.com/browse/PLNSRVCE-1311?focusedId=22426378&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-22426378) is the discussion about testing tekton results via toolchain-proxy (which is more closer to user flow) rather using tekton results SA token concept. Please let me know your thoughts on this.

Reply:
Depends how they are used to filtering I suppose @prietyc123 

Bottom line, they have been tagged for their awareness, and they are listed as reviewers.  If they are OK with this, I am.  If they are not OK, then probably adjustment is needed.  Let's see what they say.

Reply:
> However since we're running all specs labeled as "build" in build-service and specs within [this container node](https://github.com/redhat-appstudio/e2e-tests/pull/565/files#diff-420007369ea2e1a8133dc8c9a8fc032202afa2a4fc08e7f8753701ae4ab87333R156) are not related to build-service, it might make sense to stick a "pipeline" label to it, so we can filter it out on our side.

I can see the "pipeline" label has already been added to the whole [Describe](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/build/build_templates.go#L31) and neither the [when](https://github.com/redhat-appstudio/e2e-tests/pull/565/files#diff-420007369ea2e1a8133dc8c9a8fc032202afa2a4fc08e7f8753701ae4ab87333R156) or inside It spec are labeled as `buildTemplatesTestLabel`. So I think its ok to leave everything as it is and just change the required section. @psturc Did I understand you correctly?

Reply:
if by
> just change the required section

you mean to add "pipeline" label [here](https://github.com/redhat-appstudio/e2e-tests/pull/565/files#diff-420007369ea2e1a8133dc8c9a8fc032202afa2a4fc08e7f8753701ae4ab87333R156), then yes

Reply:
Done https://github.com/redhat-appstudio/e2e-tests/pull/565/commits/1a9517bd96700e9f7b960a149b4c969b35962dff

Reply:
@psturc @flacatus if now everything looks good to you. Could you please remove `hold` and add `approve` label.